src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -386,9 +386,9 @@ private function addEncodersSection(ArrayNodeDefinition $rootNode)
             ->children()
                 ->arrayNode('encoders')
                     ->example(array(
-                        'Acme\DemoBundle\Entity\User1' => 'sha512',
+                        'Acme\DemoBundle\Entity\User1' => 'bcrypt',
```

Comment:
Could we also change the deprecated `'Acme\DemoBundle\Entity\User1'` example by `'AppBundle\Entity\User1'` ? Thanks!